0-ad-history-mod-package/simulation/templates/mixins/encyclopedia/encyclopedia_hellenistic_house.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Entity>
+  <Identity>
+    <History>
+    We have more evidences houses of houses during the Hellenistic period than the structures earlier Greek culture thanks to a handbook written by the Roman architect Vitruvius Pollio. And thanks to the fact that during the Hellenistic period they started building with fired bricks rather than unfired ones.
```

Comment:
```suggestion
       We have more evidence about houses during the Hellenistic period than the structures in earlier Greek culture, thanks to a handbook written by the Roman architect Vitruvius Pollio. And thanks to the fact that during the Hellenistic period they started building with fired bricks rather than unfired ones.
```